docs/src/api/class-browsertype.md

```diff
@@ -284,6 +284,13 @@ const { chromium } = require('playwright');  // Or 'webkit' or 'firefox'.
 
 Port to use for the web socket. Defaults to 0 that picks any available port.
 
+### option: BrowserType.launchServer.path
+- `path` <[string]>
```

Comment:
Do we want to call this parameter something else to make it more explicit that it should be secret/unguessable (aside from the nodtes/comment)? Maybe `secretPath` or `unguessablePath`?

Reply:
I would call it `wsPath` and document that it should be unguessable.